### ISSUE TITLE
feat: Request to add `conventional-commit-lint` automated check for terraform-docs-samples

### ISSUE THREAD

**Issue by msampathkumar** [terraform-google-modules/terraform-docs-samples#411]
### TL;DR

Life is short, use automations. This is a feature request to add/enable following feature/check to the this terraform-docs-samples repository PRs & Issues. 

* [conventional-commit-lint](https://github.com/googleapis/repo-automation-bots/tree/main/packages/conventional-commit-lint)

This convention can be used to facilitate other automations, such as automated CHANGELOG.md updates, and automated versioning.

### Terraform Resources

```markdown
None
```

### Detailed design

Please follow the instructions provided at

* https://github.com/googleapis/repo-automation-bots/tree/main/packages/conventional-commit-lint

**Comment by bharathkkb** [comment]
I have enabled it for the repo

**Comment by glasnt** [comment]
This bot has been deprecated https://github.com/googleapis/repo-automation-bots/blob/main/packages/conventional-commit-lint/README.mdhe check is no longer working on active PRs. 

I've disabled it for the repo to ensure we have active checks against PRs.
